sensor_msgs/Temperature loop_a
sensor_msgs/Temperature loop_b
